README.md

```
@@ -65,3 +65,27 @@ To set up the project on your local environment, follow these steps:
     ```bash
     npm run lint
     ```
+
+### Assignment Details
+
+## Project Folder
+
+    Seven Folder
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)